### ISSUE TITLE
[PROPOSAL][DBT] Add a command to send OL events without running the job

### ISSUE THREAD

**Issue by sophiely** [OpenLineage/OpenLineage#2286]
**Purpose:**
Some of our users use dbt cloud to run their job but in dbt cloud you can't change the entrypoint from dbt to dbt-ol so we can't use the dbt integration.

That's why I need a command line to send OL events only, with the last generated metadata, in order to send metadata outside dbt cloud and without running the job again (I don't want to use `dbt-ol run` since the job has already run in dbt cloud and I don't want to trigger the job a second time just to send OL event).

To sum up:
DBT cloud: trigger dbt run -> Trigger web hook : to send OL events

**Proposed implementation**
We can add another command line specific to dbt-ol like `dbt-ol send-events` to send events to Open-lineage acccording to the latest metadata generated without running any dbt command.
